config/link_checker_bypass_list.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "exclude":"example.com",
+        "reason":"is not real url"
+    },
+    {
+        "exclude":"api/client/v2",
+        "reason":"will always return 400"
+    },
+    {
+        "exclude":"api/admin/v3",
+        "reason":"will always return 400"
+    },
+    {
+        "exclude":"https://data.mongodb-api.com",
+        "reason":"will always return 400"
+    },
+    {
+        "exclude":"https://cloud.mongodb.com/api/",
+        "reason":"requires authentication (returns 401)"
+    },
+    {
+        "exclude":"https://github.com/10gen/mms/blob/master/server/src/main/com/xgen/svc/mms/api/model/ApiRealmApplicationMeasurement.java",
+        "reason":"is in a hidden comment"
```

Comment:
I think the bigger reason here is that `10gen/mms` is a private repo and the checker doesn't have access so the link looks broken.